spec/mailers/course_update_email_mailer_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe CourseUpdateEmailMailer, type: :mailer do
+  let(:course) { create(:course, :with_accrediting_provider) }
+  let(:mail) { described_class.send_course_update_email(course, "name") }
+
+  before do
+    course
+    mail
+  end
+
+  context "Sending an email to a user" do
+    it "Sends an email with the correct template" do
+      expect(mail.govuk_notify_template).to eq(Settings.govuk_notify.course_update_email_template_id)
+    end
+
+    it "Sends an email to the correct email address" do
+      expect(mail.to).to eq([course.accrediting_provider.email])
+    end
+
+    it "Includes the provider name in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:provider_name]).to eq(course.provider.provider_name)
+    end
+
+    it "Includes the course name in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:course_name]).to eq(course.name)
+    end
+
+    it "Includes the course code in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:course_code]).to eq(course.course_code)
+    end
+
+    it "Includes the updated detail in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:attribute_changed]).to eq("name")
+    end
+
+    it "Includes the datetime for the detail update in the personalisation" do
+      expect(mail.govuk_notify_personalisation[:attribute_change_datetime]).to eq(course.updated_at.strftime("%k.%M on %e %B %y"))
```

Comment:
Can we hard code the expected updated date so it is easier to confirm the formatting.

Reply:
I might need a hand doing some kind of time seed thing, do you know how that would work? I know you can use timecop to freeze it, but that wouldn't mean I could put a hard string in there.